debezium-core/src/main/java/io/debezium/schema/SchemaBuilderFactory.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.schema;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+
+import io.debezium.connector.AbstractSourceInfo;
+import io.debezium.heartbeat.HeartbeatImpl;
+import io.debezium.pipeline.txmetadata.TransactionMonitor;
+import io.debezium.util.SchemaNameAdjuster;
+
+/**
+ * A factory for creating {@link SchemaBuilder} structs.
+ *
+ * @author Anisha Mohanty
+ */
+
+public class SchemaBuilderFactory {
+
+    private static final int SCHEMA_VERSION = 1;
+
+    private static final String SERVER_NAME_CLASS = "io.debezium.connector.common.ServerNameKey";
```

Comment:
It si not class name but schema name for heartbeat so please use name like `HEARTBEAT_KEY_SCHEMA_NAME`.
The other schemas should follow the same naming pattern.